<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportsWidgetBrowserStatistics</class>
 <widget class="QWidget" name="ReportsWidgetBrowserStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>379</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableView" name="browserStatisticsTableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showEntriesWithUrlOnlyCheckBox">
     <property name="text">
      <string>Only show entries that have a URL</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showAllowDenyCheckBox">
     <property name="text">
      <string>Only show entries that have been explicitly allowed or denied</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showExpired">
     <property name="text">
      <string>Show expired entries</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="tipLabel">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Double-click entries to edit.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>browserStatisticsTableView</tabstop>
  <tabstop>showEntriesWithUrlOnlyCheckBox</tabstop>
  <tabstop>showAllowDenyCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
